test/tap/tests/charset_unsigned_int-t.cpp

```diff
@@ -13,7 +13,7 @@
 int main(int argc, char** argv) {
```

Comment:
This file provoques a "stack corruption" when trying to execute the tests, this is due to [this function](https://github.com/sysown/proxysql/blob/v2.0.11/test/tap/tap/utils.cpp#L6) which is declared as non-void, but doesn't return anything, more functions in that file have the same issue.

Reply:
Fixed